Wikipedia/Code/WMFAppViewController.m

```diff
@@ -301,7 +301,7 @@ - (UINavigationController*)navigationControllerForTab:(WMFAppTabType)tab {
     return (UINavigationController*)[self.rootTabBarController viewControllers][tab];
 }
 
-- (UIViewController<WMFAnalyticsLogging>*)rootViewControllerForTab:(WMFAppTabType)tab {
+- (UIViewController<WMFAnalyticsViewNameProviding>*)rootViewControllerForTab:(WMFAppTabType)tab {
     return [[[self navigationControllerForTab:tab] viewControllers] firstObject];
```

Comment:
or blow up due to unrecognized selector, so maybe assertion isn't needed, unless we want to make that of mistake a no-op in production